sdks/python/apache_beam/runners/pipeline_context.py

```diff
@@ -124,6 +126,13 @@ def __init__(self, proto=None, default_environment=None):
           Environment(default_environment))
     else:
       self._default_environment_id = None
+    self.use_fake_coders = use_fake_coders
+
+  def coder_id_from_element_type(self, element_type):
```

Comment:
Please link to cleanup JIRA for once the workaround is no longer necessary.

Reply:
Done.